src/content/getting-started/quickstart/k3s/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+date: 2021-02-24T13:36:18+01:00
+title: "K3s"
+weight: 15
+---
+
+This quickstart guide covers the necessary steps to deploy two [K3s](https://k3s.io/) cluslters
+and deploy Submariner with Service Discovery to interconnect the two clusters. Note that this
+guide focuses on Submariner deployment on clusters with non-overlapping Pod and Service CIDRs.
+For connecting clusters with overlapping CIDRs, please refer to the
+[Globalnet guide](../../architecture/globalnet/).
+
+### Prerequisites
+
+1. Prepare two nodes with
+[K3s-supported OS](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#operating-systems)
+installed (These two nodes are referred to as node-a and node-b).
+2. Decide the Pod CIDR and the Service CIDR for each K3s cluster.
+
+### Deploy K3s Clusters
+
+#### Deploy cluster-a on node-a
+
+In this step you will deploy **cluster-a** on node-a using the the following IP ranges:
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.44.0.0/16  |10.45.0.0/16  |
```

Comment:
In  the Prerequisites, we instruct the user to choose the CIDRs but here we're telling them to use specific CIDRs.